### ISSUE TITLE
Support standard Terraform pattern of configuring providers with environment variables

### ISSUE THREAD

**Issue by sagmor** [akamai/terraform-provider-akamai#552]
It should be possible to configure the provider using only environment variables.

Every Terraform provider allows it's users to fully configure authentication using environment variables.

The approach suggested in #407 is not a good alternative because it forces every consumer to use the variables limiting flexibility.

Ideally i should be able to specify an empty provider config:

```hcl
provider "akamai" {}
```

And fully configure the authentication with Environment variables when running in an environment that doesn't allow me to easily change the filesystem while maintaining the capability to run using a local .edgerc file if available.

This would be aligned with how every other Terraform provider works (Examples):
* AWS: https://registry.terraform.io/providers/hashicorp/aws/latest/docs#environment-variables
* Azure: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs#argument-reference
* NS1: https://registry.terraform.io/providers/ns1-terraform/ns1/latest/docs#environment-variables
* Cloudflare: https://registry.terraform.io/providers/cloudflare/cloudflare/latest/docs#optional

**Comment by sagmor** [comment]
After some spelunking I managed to find out that this functionality was re-introduced in #448 and this is really a documentation problem as the environment variables listed in https://techdocs.akamai.com/terraform/docs/environment-variables should not include the `TF_VAR_` prefix.


TL;DR you CAN configure the provider with an empty config using the following environment variables:
* `AKAMAI_ACCESS_TOKEN`
* `AKAMAI_CLIENT_SECRET`
* `AKAMAI_CLIENT_TOKEN`
* `AKAMAI_HOST`

**Comment by skearney-akamai** [comment]
Thanks for bringing this to our attention. I've updated the doc to reflect this structure.
